content/page/obf-bosc-collaborationfest-2025.md

```diff
@@ -0,0 +1,27 @@
+---
+author: hmenager
+cover:
+  alt: CoFest-2022
+  image: https://www.open-bio.org/wp-content/uploads/2022/08/image4.jpg
+date: "2025-02-16T09:35:44+00:00" #creation date
+guid: https://www.open-bio.org/?page_id=7453
```

Comment:
Do we need this with Hugo, and how do we ensure the "GUID" is a globally unique ID within our website?

Reply:
I don’t think we do need those for new pages!

These are basically a hold-over for the old WP pages which had the ID-based access points in addition to the human readable ones! so for old pages it only guarantees that existing links don’t break!